examples/Docker/Makefile

```diff
@@ -0,0 +1,10 @@
+start_kafka:
```

Comment:
Please use lowercase name for directory name

Reply:
Docker -> docker
Done